core/services/chainlink/relayer_factory.go

```diff
@@ -6,8 +6,7 @@ import (
 	"fmt"
 
 	"github.com/pelletier/go-toml/v2"
-
-	"github.com/jmoiron/sqlx"
+	"github.com/smartcontractkit/sqlx"
```

Comment:
```suggestion

	"github.com/jmoiron/sqlx"
```

Reply:
I just changed this, it causes an issue with chain opts in cosmos, because  `"github.com/smartcontractkit/sqlx"` is used there.

Reply:
Should it be changed in the cosmos repo to `"github.com/jmoiron/sqlx"` then?

Reply:
It sounds like you are out of date. Everything was switched over and cosmos only uses `jmoiron`:
https://github.com/smartcontractkit/chainlink-cosmos/blob/develop/go.mod#L15

Reply:
ahh, the wonders of merging 5 different repos, I'll go on and update them now